app/services/invoice_exporter.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class InvoiceExporter
+  HEADERS = [
+    'Kdnr', 'Name1', 'Name2', 'Strasse', 'Plz', 'Ort',
```

Comment:
what is Name1 and Name2? Is that from the bookkeeping-tool?

Reply:
Yes, they requested these headers